modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreManager.java

```diff
@@ -177,19 +168,19 @@ public long allocatePage(int grpId, int partId, byte flags) throws IgniteInterna
      *
      * @param grpName Group name.
      * @param grpId Group ID.
-     * @param partitions Partition number, must be greater than {@code 0} and less {@link PageIdAllocator#MAX_PARTITION_ID} + 1.
+     * @param partitions Partition number, must be greater than {@code 0} and less {@link PageIdAllocator#MAX_PARTITION_ID}.
      * @throws IgniteInternalCheckedException If failed.
      */
     public void initialize(String grpName, int grpId, int partitions) throws IgniteInternalCheckedException {
-        assert partitions > 0 && partitions < MAX_PARTITION_ID + 1 : partitions;
+        assert partitions > 0 && partitions < MAX_PARTITION_ID : partitions;
 
         initGroupDirLock.lock(grpId);
 
         try {
             if (!groupPageStoreHolders.containsKey(grpId)) {
-                GroupPageStoreHolder<FilePageStore> holder = createGroupFilePageStoreHolder(grpName, partitions);
+                List<FilePageStore> holder = createGroupFilePageStoreHolder(grpName, partitions);
```

Comment:
this variable and the corresponding method should be renamed

Reply:
What about:
- `List<FilePageStore> holder` -> `List<FilePageStore> partitionFilePageStores`
- **createGroupFilePageStoreHolder** -> **createPartitionFilePageStores**
- **groupPageStoreHolders** -> **groupPageStores**

Reply:
Agree with the first two, but what do you mean by `groupPageStores`? Do we have "group pages"?

Reply:
Fact that groups have page stores.

Reply:
do you mean cache groups?

Reply:
There are no caches in 3.0 only tables

Reply:
what is a group then?

Reply:
Discussed personally, we apply my proposals for renaming.